index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Portfolio _one</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin />
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header>
+        <ul>
+            <li><a class="myLogo font-style" href="">My Logo</a> </li>
```

Comment:
- [ ] Amazing job by removing the `heading` tag as per the previous code reviewer's request. But as I mentioned above you did something extra which actually not correct semantically. Kindly follow the below-mentioned suggestion in order to fix the problem.

```suggestion
      <a class="myLogo font-style" href="">My Logo</a>
```